light-system-programs/tests/functional_tests.ts

```diff
@@ -84,6 +79,95 @@ describe("verifier_program", () => {
     });
   });
 
+  it("Deposit (verifier one)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY_ONE,
+      spl: true,
+      senderSpl: userTokenAccount,
+      shuffleEnabled: true,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ONE,
+    });
+  });
+
+  it("Deposit (verifier storage)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY,
+      spl: false,
+      message: Buffer.alloc(900).fill(1),
+      senderSpl: null,
+      shuffleEnabled: false,
+      verifierIdl: IDL_VERIFIER_PROGRAM_STORAGE,
+    });
+  });
+
+  it("Deposit (verifier zero)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY,
+      spl: true,
+      senderSpl: userTokenAccount,
+      shuffleEnabled: true,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ZERO,
+    });
+    const lightProvider = await Provider.init({
+      wallet: ADMIN_AUTH_KEYPAIR,
+      relayer: RELAYER,
+      confirmConfig,
+    });
+    await lightProvider.relayer.updateMerkleTree(lightProvider);
+  });
+
+  it("Withdraw (verifier zero)", async () => {
+    await performWithdrawal({
+      outputUtxos: [],
+      tokenProgram: MINT,
+      recipientSpl: recipientTokenAccount,
+      shuffleEnabled: false,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ZERO,
+    });
+  });
+
+  it("Withdraw (verifier storage)", async () => {
+    await performWithdrawal({
+      outputUtxos: [],
+      tokenProgram: SystemProgram.programId,
+      message: Buffer.alloc(900).fill(1),
+      shuffleEnabled: false,
+      verifierIdl: IDL_VERIFIER_PROGRAM_STORAGE,
+    });
+  });
+
+  it("Withdraw (verifier one)", async () => {
+    const lightProvider = await Provider.init({
+      wallet: ADMIN_AUTH_KEYPAIR,
+      relayer: RELAYER,
+      confirmConfig,
+    });
+    let user: User = await User.init({
+      provider: lightProvider,
+      account: KEYPAIR,
+    });
+    let inputUtxos: Utxo[] = [
+      user.balance.tokenBalances.get(MINT.toBase58()).utxos.values().next()
+        .value,
+    ];
+    await performWithdrawal({
+      outputUtxos: [
+        new Utxo({
+          poseidon: POSEIDON,
+          assets: inputUtxos[0].assets,
+          amounts: [BN_0, inputUtxos[0].amounts[1]],
+          assetLookupTable: lightProvider.lookUpTables.assetLookupTable,
+          verifierProgramLookupTable:
+            lightProvider.lookUpTables.verifierProgramLookupTable,
+        }),
+      ],
+      tokenProgram: MINT,
+      recipientSpl: recipientTokenAccount,
+      shuffleEnabled: true,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ONE,
+    });
```

Comment:
can you check why it is necessary to add utxos explicitly in this test but not in the other withdrawal tests?